jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/dialog.js

```diff
@@ -336,12 +337,19 @@ $(function () {
                             }
                             nextElement = jDialog.dialog.find(selectors[currentIndex]);
                             ++skipCount;
-                        } while ((nextElement.length === 0 || !nextElement.is(":visible"))
+                        } while ((nextElement.length === 0 || nextElement.is(":not(:visible)"))
                                  && skipCount <= selectors.length);
                         nextElement.focus();
                     }
                 };
-                $(v).on('keydown', func);
+
+                if($.browser.mozilla) {
+                    //To correct the problem with incorrect behaviour of autocomplete with the Tab key in Firefox.
```

Comment:
Could you please add more details for the future generation on why it's not working and how the fix helps?